docs/overrides/partials/header.html

```diff
@@ -0,0 +1,93 @@
+{#-
+    This file was automatically generated - do not edit
+  -#}
+  {% set class = "md-header" %}
+  {% if "navigation.tabs.sticky" in features %}
+    {% set class = class ~ " md-header--shadow md-header--lifted" %}
+  {% elif "navigation.tabs" not in features %}
+    {% set class = class ~ " md-header--shadow" %}
+  {% endif %}
+  <header class="{{ class }}" data-md-component="header">
+    <nav class="md-header__inner md-grid" aria-label="{{ lang.t('header') }}">
+      <a href="{{ config.extra.homepage | d(nav.homepage.url, true) | url }}" title="{{ config.site_name | e }}" class="md-header__button md-logo" aria-label="{{ config.site_name }}" data-md-component="logo">
+        {% include "partials/logo.html" %}
+      </a>
+      <label class="md-header__button md-icon" for="__drawer">
+        {% include ".icons/material/menu" ~ ".svg" %}
+      </label>
+      <div class="md-header__title" data-md-component="header-title">
+        <div class="md-header__ellipsis">
+          <div class="md-header__topic">
+            <span class="md-ellipsis">
+              <a href="/{{ config.extra.homepage | d(nav.homepage.url, true) | url }}" title="{{ config.site_name | e }}" aria-label="{{ config.site_name }}">
+                {{ config.site_name }}
+              </a>
```

Comment:
Instead of repeating, you should enclose this whole part in `</a>` end tag surrounding `{% include "partials/logo.html" %}`, i.e. move the line 14 to the line 37.

Reply:
It's not the same behavior. What you suggest would make a big clickable box. This could be annoying for phone users. If that's fine with you I'd keep it like that.